napari/_qt/range_slider/range_slider.py

```diff
@@ -405,29 +404,29 @@ def paintEvent(self, event):
         # Background
         painter.setPen(self.background_color)
         painter.setBrush(self.background_color)
-        painter.drawRect(w/2-self.slider_width/2, self.bar_width/2,
-                         self.slider_width, h-self.bar_width)
+        painter.drawRect(w / 2 - self.slider_width / 2, self.bar_width / 2,
+                         self.slider_width, h - self.bar_width)
 
```

Comment:
on breaking arguments in newlines, black is much more readable to me

Reply:
Yeah, this is one case where I have to revise my strongly-held opinion. I previously favoured "as many arguments on one line as possible", but seeing the two versions side by side, I think black's version (if some are in a new line, *all* are in a new line) is more readable.